frontend/javascripts/admin/dataset/dataset_add_remote_view.tsx

```diff
@@ -394,12 +394,14 @@ function AddRemoteLayer({
     if (userInput.startsWith("https://") || userInput.startsWith("http://")) {
       setSelectedProtocol("https");
     } else if (userInput.startsWith("s3://")) {
-      setSelectedProtocol("s3");
+      setSelectedProtocol("s3"); // Unused
```

Comment:
In case the `selectedProtocol` is set to "s3", nothing happens in the frontend code and this info is not even used for the backend call. So it is technically correct to call `setSelectedProtocol("s3");` here, but no line in this file somehow "reacts" to the protocol being s3.

Reply:
Ok, if that is really intended, maybe you could expand the comment slightly to make it explain what you just explained to me :) unused could also mean is never called, or it may indicate an open todo.
Something like `Note: the view will not change in case s3 is selected`?
I thought the authentication options are shown in different formats depending on the selected protocol. Does that mean that when s3 is selected, the same options are shown that were shown before the change? Is that what we want?